services/prefect/flows/nyt_updater.py

```diff
@@ -0,0 +1,34 @@
+from prefect import Flow
+from prefect.schedules import CronSchedule
+
+from prefect.tasks.prefect import create_flow_run, wait_for_flow_run
+from prefect.utilities.notifications import slack_notifier
+from prefect.engine.state import Failed
+
+
+def init_updater_flow():
+    with Flow(
+        "NYTParquetUpdater",
+        schedule=CronSchedule("30 6 * * *"),
+        state_handlers=[slack_notifier(only_states=[Failed])]  # notify on failure only,
+    ) as flow:
+
+        # NYTimesCasesDeaths and UpdateParquetFiles flows must already be registered via
+        # generated_flows.py and update_api_view.py
+        nyt_flow = create_flow_run(
+            flow_name="NYTimesCasesDeaths", project_name="can-scrape"
+        )
+        wait_for_nyt = wait_for_flow_run(nyt_flow, raise_final_state=True)
```

Comment:
Out of curiosity, what would happen if NYT was already running and so when we try to create the flow here it ends up skipping it?

Would we immediately go ahead and create the parquet file (without up-to-date NYT data)?  Or would this flow fail?  Or... what?

Reply:
Good question--I'm not sure... I'll look in the docs/do some testing to find out. 

Ultimately, we would probably want to cancel the existing instance and have this one run in case the previous run had started before the data was updated (but I'm not immediately sure how that might be done).